go/libkb/kbsig.go

```diff
@@ -771,6 +772,14 @@ func WalletProof(me *User, walletAddress keybase1.StellarAccountID,
 	walletSection.SetKey("address", jsonw.NewString(walletAddress.String()))
 	walletSection.SetKey("network", jsonw.NewString(string(WalletNetworkStellar)))
 
+	// Inner links can be hidden. To prevent an attacker from figuring out the
+	// contents from the hash of the inner link, add 18 random bytes.
+	entropyBytes, err := RandBytes(18)
+	if err != nil {
+		return nil, fmt.Errorf("failed to generate entropy bytes: %v", err)
+	}
+	walletSection.SetKey("entropy", jsonw.NewString(base64.StdEncoding.EncodeToString(entropyBytes)))
```

Comment:
is this the same number of bytes, and the same encoding  as we use in teams?

Reply:
yep
[makeSCTeamEntropy](https://github.com/keybase/client/blob/master/go/teams/sig.go#L236)
[slot](https://github.com/keybase/client/blob/master/go/teams/sig.go#L86)